var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -140,6 +142,21 @@ class Gromacs(CMakePackage):
         when="+cp2k",
         msg="GROMACS and CP2K should use the same blas, please disable bundled blas",
     )
+    conflicts(
+        "%intel",
+        when="@2022:",
+        msg="GROMACS %intel support was removed in version 2022",
```

Comment:
@mabraham is that right?

Reply:
This seems right from comparing [2021 docu](https://manual.gromacs.org/2021/install-guide/index.html) and [2022 docu](https://manual.gromacs.org/2022/install-guide/index.html).

Reply:
Yes, the Intel classic compiler is no longer supported by GROMACS. The LLVM-based ICPX/DPCPP compiler is supported by GROMACS.